app/views/lessons/previews/show.html.erb

```diff
@@ -10,18 +10,23 @@
         <div data-controller='tabs share-preview' data-share-preview-url-value="<%= lessons_preview_share_path(format: :turbo_stream) %>" data-tabs-index='0' data-tabs-active-class="text-gray-700 bg-gray-300/50 hover:bg-gray-300 dark:bg-gray-700/90" data-tabs-inactive-class="hidden">
           <div class="flex justify-between">
             <div class="flex items-center mb-3">
-              <button data-action="tabs#change" class="text-gray-600 bg-gray-300/50 hover:text-gray-800 hover:bg-gray-200 dark:bg-gray-700/40 dark:text-gray-300 rounded-md border border-transparent px-3 py-1.5 font-medium cursor-pointer focus:outline-none" data-tabs-target='tab'>
+              <button data-action="tabs#change" class="text-gray-600 bg-gray-300/50 hover:text-gray-800 hover:bg-gray-200 dark:bg-gray-700/40 dark:text-gray-300 dark:hover:text-gray-200 dark:hover:bg-gray-600 rounded-md border border-transparent px-3 py-1.5 font-medium cursor-pointer focus:outline-none" data-tabs-target='tab'>
                 Write
               </button>
-              <button data-action="tabs#change" class="ml-2 text-gray-600 hover:text-gray-800 hover:bg-gray-200 dark:bg-gray-700/40 dark:text-gray-300 rounded-md border border-transparent px-3 py-1.5 font-medium cursor-pointer focus:outline-none" data-tabs-target='tab'>
+              <button data-action="tabs#change" class="ml-2 text-gray-600 hover:text-gray-800 hover:bg-gray-200 dark:bg-gray-700/40 dark:text-gray-300 dark:hover:text-gray-200 dark:hover:bg-gray-600 rounded-md border border-transparent px-3 py-1.5 font-medium cursor-pointer focus:outline-none" data-tabs-target='tab'>
                 Preview
               </button>
             </div>
-
-            <%= button_to '#', class: 'hidden button button--secondary py-2 px-3', method: :get, data: { share_preview_target: 'button', action: 'share-preview#share' } do %>
-              <%= inline_svg_tag 'icons/share.svg', class: 'h-3 w-3 mr-2', aria: true, title: 'share', desc: 'share icon' %>
-              Share
-            <% end %>
+            <div class="flex items-center mb-3">
+              <%= button_to lessons_preview_path, class: 'hidden button button--secondary py-2 px-3', method: :get, data: { share_preview_target: 'button' } do %>
```

Comment:
Using the share controller for the clear button could be a little confusing and prohibitive in the future if we need to expand out what share does.

I can see the reason why we'd want to hide it when theres no content, but with this being a non user facing tool, I'd be very tempted to keep it simple and just have the clear button always displayed. 

The share button creates a record behind the scenes and will raise an error if theres no content, but theres close to no cost with the get request the clear button triggers.

```suggestion
              <%= button_to lessons_preview_path, class: 'button button--secondary py-2 px-3', method: :get do %>
```

Reply:
Sure, we can do that, personally I hate when UI elements that I see move, it disorients me so this is why I went for this very cheap and dirty way of sharing the hiding behaviour.